pep/app/pods/search/read/controller.ts

```diff
@@ -218,6 +218,18 @@ export default class SearchRead extends Controller {
             this.searchHitNumber = this.searchHitNumber - 1;
         }
     }
+
+    /**
+     * Set the search hit number to the next number. Most likely its the same number they were on, so make it undefined first
+     *
+     * @param {number} number
+     * @memberof SearchRead
+     */
+    @action
+    viewSearchHitNumber(number: number) {
+        this.searchHitNumber = undefined;
+        this.searchHitNumber = number;
```

Comment:
I dont _think_ this will have any actual effect on ember/glimmer, as it waits until the end of the runloop to coalesce all value changes before a new render. its more likely in these cases you want to do something like:
```ts
this.searchHitNumber = undefined;
next(this, () => this.searchHitNumber = number);
```